runtime/doc/lua.txt

```diff
@@ -3494,11 +3497,7 @@ Iter:nth({n})                                                     *Iter:nth()*
         any
 
 Iter:nthback({n})                                             *Iter:nthback()*
-    Return the nth value from the end of the iterator.
-
-    This function advances the iterator.
-
-    Only supported for iterators on list-like tables.
+    Gets the nth value from the end of a |list-iterator| (and advances to it).
```

Comment:
@gpanders would it be too clever if we supported negative `n` for `nth()`, then we could eliminate `nthback()` ?

Reply:
That sounds fine to me